resources/lang/en/model_validation.php

```diff
@@ -88,11 +88,28 @@
         'invalid_email' => "Doesn't seem to be a valid email address.",
         'too_short' => 'New password is too short.',
         'unknown_duplicate' => 'Username or email address already used.',
+        'username_available_in_days' => '{1} This username will be available for use in :count day.|[2,*] This username will be available for use in :count days.',
+        'username_available_in_hours' => '{1} This username will be available for use in :count hour.|[2,*] This username will be available for use in :count hours.',
```

Comment:
does it actually do anything? Looks the same as without explicit numbers.

Reply:
it makes the `pl` translation use the `en` rules when untranslated

Reply:
looks more like symfony/laravel bug ಠ_ಠ also missing 0.

Reply:
we won't need 0 yet :trollface:

Reply:
I'd rather "fix" the `trans_choice` function so we don't need to do this for everything.